include/IECoreGL/Camera.h

```diff
@@ -57,22 +57,25 @@ class IECOREGL_API Camera : public Renderable
 		/// then it'll be calculated to be -1,1 in width and to preserve
 		/// aspect ratio (based on resolution) in height.
 		Camera( const Imath::M44f &transform = Imath::M44f(),
+			bool orthographic = true,
 			const Imath::V2i &resolution = Imath::V2i( 640, 480 ),
-			const Imath::Box2f &screenWindow = Imath::Box2f(),
+			const Imath::Box2f &normalizedScreenWindow = Imath::Box2f( Imath::V2f(-1), Imath::V2f(1)),
 			const Imath::V2f &clippingPlanes = Imath::V2f( 0.1, 1000 )
 		);
 
 		IE_CORE_DECLARERUNTIMETYPEDEXTENSION( IECoreGL::Camera, CameraTypeId, Renderable );
 
+		void render( State *currentState ) const override;
+
 		/// Specifies the transform of the camera relative to the world.
 		void setTransform( const Imath::M44f &transform );
 		const Imath::M44f &getTransform() const;
 
 		void setResolution( const Imath::V2i &resolution );
 		const Imath::V2i &getResolution() const;
 
-		void setScreenWindow( const Imath::Box2f &screenWindow );
-		const Imath::Box2f &getScreenWindow() const;
+		void setNormalizedScreenWindow( const Imath::Box2f &normalizedScreenWindow );
```

Comment:
This is the screen window at a distance of 1 from the camera.  It's a convenient way to characterize the shape of the frustum, independent of the focal length.  I've definitely found this a very useful method, but I'm open to suggestions on the name.

Maya actually calls this getFrustum ( well, they have two separate getViewingFrustum and  getRenderingFrustum ) - would you prefer getFrustum?  The problem is that a lot of package would use getFrustum to return a set of planes in 3D space, but I guess we could document that getFrustum returns a normalized screenWindow.

Reply:
If I recall correctly, the RenderMan/Cortex field of view corresponded to the ( -1, 1 ) range on the image plane, and the screen window was measured with respect to that. Correct? I suppose it will be a pain to continue with that definition?

I suspect a prefix alone isn't going to be sufficient to document this either way, so my vote would be to drop the prefix and then document it.

Reply:
The point of this is to have a maximally succinct way of defining the frustum that doesn't depend on anything else.

I would definitely find it clearer to put something in the name that makes it clear "this isn't the same screenWindow you're used to, you should check the docs".  Naming it identically to something that means something else in other packages seem unnecessarily confusing, but I don't feel particularly strongly.  I can rename it to "screenWindow".